proprietary/tokens/src/brand/ams/typography.tokens.json

```diff
@@ -1,12 +1,29 @@
 {
   "ams": {
-    "text": {
+    "typography": {
       "font-family": { "value": "'Amsterdam Sans', Arial, sans-serif" },
-      "font-weight": {
-        "normal": { "value": 400 },
-        "bold": { "value": 800 }
+      "body-text": {
+        "font-size": { "value": "clamp(1.125rem, calc(1.0313rem + 0.4688vw), 1.5rem)" },
+        "font-weight": { "value": "400" },
+        "line-height": { "value": "1.6" },
+        "sm": {
+          "font-size": { "value": "clamp(0.9643rem, calc(0.9054rem + 0.2946vw), 1.2rem)" },
+          "line-height": { "value": "1.6" }
+        },
+        "lg": {
+          "font-size": { "value": "clamp(1.3125rem, calc(1.1719rem + 0.7031vw), 1.875rem)" },
+          "line-height": { "value": "1.5" }
+        },
+        "xl": {
+          "font-size": { "value": "clamp(1.5313rem, calc(1.3281rem + 1.0156vw), 2.3438rem)" },
+          "line-height": { "value": "1.3" }
+        },
+        "bold": {
+          "font-weight": { "value": "700" }
+        }
       },
-      "level": {
+      "heading": {
+        "font-weight": { "value": "800" },
         "0": {
           "font-size": { "value": "clamp(2.4316rem, calc(1.8951rem + 2.6826vw), 4.5776rem)" },
           "line-height": { "value": "1.15" }
```

Comment:
I can't comment on the correct line, but: We don't have Heading level 5 and 6, shouldn't we remove them from this list?

Reply:
We do have them visually, through the `size` prop. The class names `.ams-heading--level-5` and `.ams-heading--level-6` use these tokens.